src/Storefront/Resources/app/storefront/src/scss/component/_filter-range.scss

```diff
@@ -44,7 +44,7 @@ This filter item should only be used inside a filter panel.
     }
 }
 
-.filter-range-currency-symbol {
+.filter-range-unit {
```

Comment:
The renaming of the CSS selector is a breaking change. You must deprecate the old class.

```
/* ----------------------------------------------------------------------------------------------------------
   * @deprecated tag:v6.5.0 - `filter-range-currency-symbol` will be removed, use `filter-range-unit` instead
* --------------------------------------------------------------------------------------------------------- */
.filter-range-currency-symbol,
.filter-range-unit {
```

For general information on how to ensure backward compatibility, see this [link](https://developer.shopware.com/docs/resources/guidelines/code/backward-compatibility#storefront).